### ISSUE TITLE
The focus ring is not correctly displayed on the Report button

### ISSUE THREAD

**Issue by rleitan** [mozilla/voice-web#2099]
[Environment]:
Mac 10.14 - Safari

[STR]:
1.Open https://voice.allizom.org
2.Go to Contribute (Listen or Speak)
3.Press Tab key to navigate through buttons

[Expected result]:
The focus ring should be correctly displayed on Report button
e.g.
<img width="203" alt="Screenshot 2019-07-09 at 16 29 29" src="https://user-images.githubusercontent.com/41474886/60892882-8dc04f80-a268-11e9-94b9-343a44b00d53.png">

[Actual result]: 
The focus ring is not correctly displayed on the Report button

<img width="225" alt="Screenshot 2019-07-09 at 16 29 13" src="https://user-images.githubusercontent.com/41474886/60892837-7a14e900-a268-11e9-8ce0-9a3eadd11183.png">
